same_site config setting needs to be changed from a bool

Start a new project with `jetzig init` then run `jetzig bundle`


```
+- install test
   +- zig build-exe test ReleaseFast x86_64-linux 2 errors
src/main.zig:90:14: error: expected type '?jetzig.http.Cookies.SameSite', found 'bool'
            .same_site = true,
```

It appears that [6e6f1be](https://github.com/jetzig-framework/jetzig/commit/6e6f1bec1ba6c315972a20e168632783227115bf) changed the API but didn't change the generated `src/main.zig`  that the `jetzig` CLI tool creates.

Changing it to be

```diff
index 26f638a..0b9402e 100644
--- a/src/main.zig
+++ b/src/main.zig
@@ -87,7 +87,7 @@ pub const jetzig_options = struct {
             .path = "/",
         },
         .production => .{
-            .same_site = true,
+            .same_site = .strict,
```

Gets past that compilation error but then hits a different error that I am not skilled enough to fix (Zig compiler bug?)

```
+- install <project>
   +- zig build-exe <project> ReleaseFast x86_64-linux 1 errors
/snap/zig/13835/lib/std/crypto/aes/soft.zig:824:1: error: emit MIR failed: CannotEncode (Zig compiler bug)
fn table_lookup(table: *align(64) const [4][256]u32, idx0: u8, idx1: u8, idx2: u8, idx3: u8) [4]u32 {
```

This was all tested today with jetzig b8af924 and Zig `0.14.0`

--- a/demo/src/main.zig
+++ b/demo/src/main.zig
@@ -87,7 +87,7 @@ pub const jetzig_options = struct {
             .path = "/",
         },
         .production => .{
-            .same_site = true,
+            .same_site = .lax,
             .secure = true,
             .http_only = true,
             .path = "/",

